parser.js

```diff
@@ -99,11 +100,14 @@ function parse(code, options) {
          */
         if (typeof options.loggerFn === "function") {
             extra.log = options.loggerFn;
+        } else if (options.loggerFn === false) {
+            /** */ // eslint-disable-line valid-jsdoc
```

Comment:
If I don’t add the JSDoc comment, it errors at line 98 instead. I’ll switch to `Function.prototype`.